### PR TITLE
added db migration script and migration script for sf migration

### PR DESCRIPTION
**Pull request by aakashuipath**

### CODE REVIEW

File: platform/sfmigration/dbmigrationbcp.bash [hunk 1]
```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+function update_database_table(){
```
Comment (SahilChaudhary25) [inline platform/sfmigration/dbmigrationbcp.bash:3]: Please add function docs along with parameter passed description

File: platform/sfmigration/dbmigrationbcp.bash [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+export SRC_PKGMANAGER_DB_PASSWORD=$(cat $CREDENTIALS_FILE | jq -r 'select(.SRC_PKGMANAGER_DB_PASSWORD != null) | .SRC_PKGMANAGER_DB_PASSWORD')
+
+export SRC_TRAINER_DB_NAME=$(cat $CREDENTIALS_FILE | jq -r 'select(.SRC_TRAINER_DB_NAME != null) | .SRC_TRAINER_DB_NAME')
+
+export SRC_TRAINER_DB_SCHEMA=$(cat $CREDENTIALS_FILE | jq -r 'select(.SRC_TRAINER_DB_SCHEMA != null) | .SRC_TRAINER_DB_SCHEMA')
+
+export SRC_TRAINER_DB_USERNAME=$(cat $CREDENTIALS_FILE | jq -r 'select(.SRC_TRAINER_DB_USERNAME != null) | .SRC_TRAINER_DB_USERNAME')
+
+export SRC_TRAINER_DB_PASSWORD=$(cat $CREDENTIALS_FILE | jq -r 'select(.SRC_TRAINER_DB_PASSWORD != null) | .SRC_TRAINER_DB_PASSWORD')
```
Comment (SahilChaudhary25) [inline platform/sfmigration/dbmigrationbcp.bash:84]: Lets break code into smaller function and make it modular with docs

File: platform/sfmigration/input.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (SahilChaudhary25) [inline platform/sfmigration/input.json:1]: platform/sfmigration/input.json -> platform/sfmigration/sampleinput.json
Please star sdding readme file with detailed steps

File: platform/sfmigration/mastermigrationscript.bash [hunk 4]
```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+: '
+This scipt will export all data stored in blob storage from target environments.
```
Comment (SahilChaudhary25) [inline platform/sfmigration/mastermigrationscript.bash:4]: Looks wrong

**Review by SahilChaudhary25 — CHANGES_REQUESTED** [review]

File: platform/sfmigration/mastermigrationscript.bash (hunk 4 above)
Comment (aakashuipath) [reply platform/sfmigration/mastermigrationscript.bash]: done

File: platform/sfmigration/dbmigrationbcp.bash (hunk 1 above)
Comment (aakashuipath) [reply platform/sfmigration/dbmigrationbcp.bash]: done

**Pull request closed without merging**